anthos-bm-utils/abm-nettest/nettest.yaml

```diff
@@ -33,14 +33,18 @@ spec:
     spec:
       containers:
       - name: echoserver
-        image: gcr.io/anthos-baremetal-release/simplehttpserver:v0.3.3__linux_amd64
+        image: gcr.io/anthos-baremetal-release/simplehttpserver:v0.3.7-gke.9__linux_amd64
         imagePullPolicy: IfNotPresent
+        args: ["--port=9222"]
         ports:
-        - containerPort: 8080
+        - containerPort: 9222
```

Comment:
Whats the context for the port update

Reply:
This is the reduce the likelyhood of port collision (mainly for hostnetwork echoserver/cloudprobers)